dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ContentletIndexAPIImplTest.java

```diff
@@ -557,7 +557,7 @@ public void removeContentFromIndexByStructureInode () throws Exception {
 
             int x=0;
             do {
-                Thread.sleep(200);
+                Thread.sleep(5000);
```

Comment:
is there a way to fix this test without using a sleep?

Reply:
This test was not related to the PP filters, but it fails sporadically in master. When I tried to debug it, it succeeded. So I increase the sleep as a temporary fix (did not want to spend a lot of time here since it's not related to the task), this test is in the doc of tests that fail sporadically so when all those tests are attacked we can take another look to search another approach.